README.md

```diff
@@ -2,7 +2,7 @@
 
 The **AWS SDK for Swift** is a pure Swift SDK for accessing any & all AWS services.
 
-**The AWS SDK for Swift is currently in developer preview and is intended strictly for feedback purposes only. Do not use this SDK for production workloads. Refer to the SDK [stability guidelines](docs/stability.md) for more detail.**
+**The AWS SDK for Swift is generally available and approved for production workloads. Refer to the SDK [stability guidelines](docs/stability.md) for more detail.**
```

Comment:
Link "stability guidelines" to https://sdk.amazonaws.com/swift/api/awssdkforswift/latest/documentation/awssdkforswift/stability